INSTALL.md

```diff
@@ -6,7 +6,7 @@ Qiita is pip installable, but depends on some non-python packages that must be i
 Install the non-python dependencies
 -----------------------------------
 
-* [PostgreSQL](http://www.postgresql.org/download/) (we have tested most extensively with 9.3)
+* [PostgreSQL](http://www.postgresql.org/download/) (minimum required version 9.3.0, we have tested most extensively with 9.3)
```

Comment:
Added a note to CHANGELOG.md.
The reason is that we are now using the SQL type JSON, which was introduced in postgres 9.3.0